solarforecastarbiter/reports/figures/bokeh_datamodel.py

```diff
@@ -0,0 +1,55 @@
+"""File containing datamodel objects for creating bokeh metric
```

Comment:
Why the separate module? We won't be able to just delete the file when we want to remove bokeh, and it seems confusing to me